### ISSUE TITLE
Broken anchor links

### ISSUE THREAD

**Issue by koalie** [w3c/aria#1229]
https://github.com/evcohen/eslint-plugin-jsx-a11y/pull/676/files

via https://lists.w3.org/Archives/Public/site-comments/2020Mar/0007.html

**Comment by joanmarie** [comment]
Silly question: But what are we supposed to do in our spec to address this?

**Comment by ljharb** [comment]
Add redirects, so the old links redirect to the changed locations, per your spec's governing style: https://www.w3.org/Provider/Style/URI

**Comment by joanmarie** [comment]
I didn't realize we had changed those URIs, but we can take a look.

**Comment by michael-n-cooper** [comment]
It looks like the commenter bookmarked WAI-ARIA 1.0, using the "latest version URI" at the time, https://www.w3.org/TR/wai-aria/. In 1.1 and later we changed the structure. This is allowed, but when the latest version URI was updated with the release of ARIA 1.1, it started to reference a new file structure, rendering old links obsolete.

I'm not sure this is technically a violation of our site policy, as bookmarks to the old dated URI would still work. But the use of latest version URIs in TR does mean this can be a problem. I should be able to put redirects in place, but will have to make arrangements with the webmaster.

**Comment by deniak** [comment]
@michael-n-cooper yes, this unfortunately happens when there are files and anchors changes between different versions of the same specification.
In the ideal world, we would automate all these redirects but we first need to track that kind of changes which can be tricky. I know @plehegar was looking at this with some editors but I'm not sure what's the status of this project.
That being said, if you can provide the list of redirects, I'll deploy them.

**Comment by pkra** [comment]
@daniel-montalvo I think this was fixed (but the link in the top card no longer works).

**Comment by daniel-montalvo** [comment]
I'd say so @pkra . We no longer have a way to trace this unfortunately.

So I guess we'd be OK with W3C [link checker](https://validator.w3.org/checklink) to avoid these things from happening in the future.

**Comment by ljharb** [comment]
The top link is now https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/pull/676/files

**Comment by pkra** [comment]
Thanks @ljharb!

@daniel-montalvo can you take another look?
